src/screens/settings/settings.tsx

```diff
@@ -160,6 +162,26 @@ export class SettingsScreenComponent extends React.Component<
                             <Icon name="chevron-right" size={16} style={styles.icon} />
                         </View>
                     </TouchableOpacity>
+
+                    <View style={styles.divider} />
+
+                    <TouchableOpacity
+                        style={styles.rowContainer}
+                        onPress={() =>
+                            this.setState({ changePIN: true }, () => {
+                                this.passwordModal.requestPassword().then(() => {
+                                    // disable changePIN if you want to reattempt to change the PIN code
+                                    this.setState({ changePIN: false }, () =>
+                                        Dialog.info(translate('Settings.successChangePin'), '')
+                                    );
```

Comment:
@NovacAndrei  - ce se intampla dupa ce ai schimbat pinu cu wallet-urile? - ai facut o tranzactie sau un switch de wallet?

Reply:
Working now on this, using `readEncrypted` and `storeEncrypted`

Reply:
Done